pkg/router/template/plugin_test.go

```diff
@@ -1162,20 +1162,6 @@ func TestHandleRouteUpgradeValidation(t *testing.T) {
 			},
 			unservableInFutureVersionsExpected: false,
 		},
-		{
-			name: "SHA1 certificate should be unservable in future versions",
-			route: &routev1.Route{
-				Spec: routev1.RouteSpec{
-					Host: "www.reencrypt.sha1.test",
-					TLS: &routev1.TLSConfig{
-						Termination: routev1.TLSTerminationReencrypt,
-						Certificate: testCertificateRsaSha1,
-						Key:         testCertificateRsaSha1Key,
-					},
-				},
-			},
-			unservableInFutureVersionsExpected: true,
-		},
```

Comment:
Why are you removing the test case, as opposed to flipping the expectation?
```suggestion
		{
			name: "SHA1 certificate should be unservable in future versions",
			route: &routev1.Route{
				Spec: routev1.RouteSpec{
					Host: "www.reencrypt.sha1.test",
					TLS: &routev1.TLSConfig{
						Termination: routev1.TLSTerminationReencrypt,
						Certificate: testCertificateRsaSha1,
						Key:         testCertificateRsaSha1Key,
					},
				},
			},
			unservableInFutureVersionsExpected: false,
		},
```

Reply:
Oh yes, we should just flip the expectation instead of removing it. Done.